utils/lib/parse_timing.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+import re
```

Comment:
We should just fix VPR to produce a machine readable output file here.

Reply:
That's true. Still even with eg. a JSON file with usage and timing report there would be a need for a script that compares the data against a reference

Reply:
Filed a feature request for enabling machine readable output: https://github.com/verilog-to-routing/vtr-verilog-to-routing/issues/1944

Reply:
Opened PR in vtr that implements machine readable summary outputs for timings and block usage: https://github.com/verilog-to-routing/vtr-verilog-to-routing/pull/1947

Reply:
https://github.com/SymbiFlow/symbiflow-arch-defs/pull/2354/commits/415992bf567e2c865fb20e94e705c71461ee8ce8 modifies ASSERT_TIMING scripts (report_timing.py, parse_timing.py) and cmake flow to use VPR JSON output instead of VPR stdout for acquiring timing information.
This commit is expected to fail in CI on unknown VPR options (`--write_block_usage`, `--write_timing_summary`) as they are yet to be introduced to VPR in https://github.com/verilog-to-routing/vtr-verilog-to-routing/pull/1947
CI will go green when VPR PR will be merged, SymbiFlow VPR package will be updated and PR code will be rebased onto newest master.